### ISSUE TITLE
Wrong nesting of HTML elements

### ISSUE THREAD

**Issue by DanJFletcher** [bennyhusband/bennyhusband.github.io#1]
Hey, it's Dan from freeCodeCamp London.

Great start to your portfolio site!

A quick glance at your HTML I noticed a few mistakes:

Your HTML should be structured like this
```html
<!DOCTYPE html>
<html> <!-- should only be one opening html tag -->
  <head> <!-- also only one opening head tag -->
    <!-- links to css, meta tags, occasionally links to JavaScript goes here -->
  </head> <!-- close the head section before the opening <body> tag -->

  <body> <!-- Only one opening body tag -->
    <!-- All other content goes here including, links to JavaScript files (CSS is always going to be
    inside your head tags) -->

    <!-- A footer tag is no different than any other normal HTML tag - it is 
    basically the same as a <div> only it has "symantical" meaning. Which just means, 
    search engines like google have some context into what this element actually is -->
    <footer>This is some content</footer>

  </body> <!-- Notice we close the body after all content - including that footer tag -->
</html> <!-- Of course, we need to close our html tag at the very end -->

```

In your code, I noticed you have multiple `<head>` sections, some content tags outside of the `<body>` section, and more than one body tag as well, plus some extra closing `</html>` tags.

Let me know if you want any help tweaking things - and thanks for sharing your site with us on the FCC group!

**Comment by bennyhusband** [comment]
Thank you so much for taking the time to do this. Right now I am struggling to wrap my head around JS, and I am worrying I will forget everything I learned about HTML and CSS, maybe this is a good time to break from JS and make the corrections to my portfolio site.
